### ISSUE TITLE
Make venue information easier to find, including accessibility information

### ISSUE THREAD

**Issue by AmeliaBR** [Maps4HTML/Maps4HTML-Workshop-2020#7]
(Pending finalization of the venue details, of course!)

This should be easy to find from the top-level menu & should link out to any information for the OGC meetings, when they are available, and any information about directions/parking/transit/accessibility from the host facility.

Accessibility information to consider:

- Is venue step-free / wheelchair accessible?
- What about washrooms?
- Captioning (probably just IRC minutes unless a sponsor steps up)
- Allergy/dietary accommodation (assuming some budget for catering)

**Comment by AmeliaBR** [comment]
From @prushforth:
[Standard Venue Arrangements for W3C Events: Conferences, Workshops, and Meetings](https://www.w3.org/2011/06/WorkshopHost.html)

**Comment by AmeliaBR** [comment]
I'm currently linking to the venue website, but they don't have details about access:
https://www.cae.com/civil-aviation/locations/cae-montreal/

**Comment by AmeliaBR** [comment]
Closing as no longer relevant. (😞)

We might still need some (software) accessibility info as part of the [tools / information for participants section](https://maps4html.org/Maps4HTML-Workshop-2020/agenda#tools), but that is pending on decisions about those tools.
